agir/carte/components/common/Map/ClickableMap.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import Map from "@agir/carte/common/Map";
+import PropTypes from "prop-types";
+import {useMobileApp} from "@agir/front/app/hooks";
+
+const ClickableMap = (props) => {
+  const { location, zoom, subtype, center } = props;
+  const isIOS = useMobileApp().isIOS;
+  const isAndroid = useMobileApp().isAndroid;
+  let latitude = location.coordinates.coordinates.toString().split(",")[1];
+  let longitude = location.coordinates.coordinates.toString().split(",")[0];
```

Comment:
Tu as besoins de faire comme ça ? Ça ne suffit pas juste de faire :
```suggestion
  const [ longitude, latitude ] = location.coordinates.coordinates;
```